index.js

```diff
@@ -24,6 +24,7 @@ const UtilityToken = require('./src/ContractInteract/UtilityToken');
 const Mosaic = require('./src/Mosaic');
 
 const Utils = require('./src/utils/Utils');
+const ProofGenerator = require('./src/utils/Proof');
```

Comment:
I like to order the dependencies alphabetically in each block so that it is easier to find them. But that is not mandatory, of course. Just a suggestion.